iothub/device/tests/IotHubDeviceClientTests.cs

```diff
@@ -267,19 +267,17 @@ await deviceClient
                 .ConfigureAwait(false);
 
             // assert
-            await innerHandler
-                .Received()
-                .DisableMethodsAsync(Arg.Any<CancellationToken>())
-                .ConfigureAwait(false);
+            innerHandler
```

Comment:
I'm not sure what Verify does.

Reply:
Shouldn't there be a `Times` parameter or something?

Reply:
I think Verify defaults to at least once if no Times parameter is passed in

Reply:
I'd rather the code be explicit.